src/main/java/Wallybot.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+// Import Task class
+import tasks.Task;
+
+// Import custom exceptions
+import exceptions.InvalidCommandException;
+// Description exceptions
+import exceptions.descriptions.EmptyDescriptionException;
+import exceptions.descriptions.WrongFormatDescriptionException;
+import exceptions.descriptions.IncompleteDescriptionException;
+
+
+/**
+ * Wallybot is a personal assistant chatbot.
+ * It keeps track of your tasks and allows you to manage them.
+ */
+public class Wallybot {
+    // Divider to format text for better user readability
+    private static final String DIVIDER = "- - - - - - - - - - - - -";
+
+    /**
+     * Initialise chatbot
+     */
+    public static void initWally() {
+        String logo =
+            " __       __          __ __          __                  __     \n" +
+            "|  \\  _  |  \\        |  \\  \\        |  \\                |  \\    \n" +
+            "| ▓▓ / \\ | ▓▓ ______ | ▓▓ ▓▓__    __| ▓▓____   ______  _| ▓▓_   \n" +
+            "| ▓▓/  ▓\\| ▓▓|      \\| ▓▓ ▓▓  \\  |  \\ ▓▓    \\ /      \\|   ▓▓ \\  \n" +
+            "| ▓▓  ▓▓▓\\ ▓▓ \\▓▓▓▓▓▓\\ ▓▓ ▓▓ ▓▓  | ▓▓ ▓▓▓▓▓▓▓\\  ▓▓▓▓▓▓\\\\▓▓▓▓▓▓  \n" +
+            "| ▓▓ ▓▓\\▓▓\\▓▓/      ▓▓ ▓▓ ▓▓ ▓▓  | ▓▓ ▓▓  | ▓▓ ▓▓  | ▓▓ | ▓▓ __ \n" +
+            "| ▓▓▓▓  \\▓▓▓▓  ▓▓▓▓▓▓▓ ▓▓ ▓▓ ▓▓__/ ▓▓ ▓▓__/ ▓▓ ▓▓__/ ▓▓ | ▓▓|  \\\n" +
+            "| ▓▓▓    \\▓▓▓\\▓▓    ▓▓ ▓▓ ▓▓\\▓▓    ▓▓ ▓▓    ▓▓\\▓▓    ▓▓  \\▓▓  ▓▓\n" +
+            " \\▓▓      \\▓▓ \\▓▓▓▓▓▓▓\\▓▓\\▓▓_\\▓▓▓▓▓▓▓\\▓▓▓▓▓▓▓  \\▓▓▓▓▓▓    \\▓▓▓▓ \n" +
+            "                           |  \\__| ▓▓                           \n" +
+            "                            \\▓▓    ▓▓                           \n" +
+            "                             \\▓▓▓▓▓▓                            \n"
+        ;
+        String chatbotName = "Wallybot";
+
+        System.out.println(logo);
+        System.out.println("Beep boop. Hello! I'm " + chatbotName + "! :P");
+        System.out.println("What can I do for you today?");
+        System.out.println(DIVIDER);
+    }
+
+    /**
+     * Exit Wallybot
+     */
+    public static void exitWally() {
+        System.out.println("Productive day today! :D Shutting down...");
+    }
+
+    /**
+     * Echo user input
+     */
+    public static void echo(String input) {
+        System.out.println(input);
+    }
+
+
+    /*  MAIN  */
+    public static void main(String[] args) {
+        // Initialise chatbot
+        initWally();
+        Scanner in = new Scanner(System.in);
+
+        // Read user input continuously
+        scan: while (true) {
```

Comment:
try sticking to the coding standard for while () loops

https://se-education.org/guides/conventions/java/basic.html#layout